docs/guides/admin/docs/css/extra.css

```
@@ -160,6 +160,13 @@ a.header-link:hover {
     opacity: 1;
 }
 
+details {
+  border-left: 2px solid silver;
+  padding: 5px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+}
+
```

Looks good! And while we are at it, I suggest the following addition. This adds a little chevron using the glyphicon font.

```suggestion
details {
  border-left: 2px solid silver;
  padding: 5px;
  margin-bottom: 10px;
  margin-left: 10px;
}

details summary::after {
  position: relative;
  top: 1px;
  font-family: 'Glyphicons Halflings';
  font-style: normal;
  font-weight: 400;
  line-height: 1;
  padding-left: 1ex;
  content: '\e114'; /* glyphicon-chevron-down*/
}

details[open] summary::after {
  content: '\e113'; /* glyphicon-chevron-up*/
}
```

---

I'd rather not do this here because I plan to merge this without waiting for the Technical Meeting, since it's basically just a (partial) revert of an earlier PR today. Could you file a new PR for this?

---

yes, will do

---

See #5368